test/e2e/v1beta1/scripts/gh-actions/run-notebook.sh

```diff
@@ -0,0 +1,71 @@
+#!/bin/bash
+
+# Copyright 2024 The Kubeflow Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This bash script is used to run the example notebooks
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+NOTEBOOK_INPUT=""
+NOTEBOOK_OUTPUT="-" # outputs to console
+NAMESPACE="default"
+TRAINING_PYTHON_SDK="./sdk/python"
```

Comment:
This should be, Katib SDK.

Reply:
@yehudit1987 We should still pass the SDK to the Notebooks, so we can build it from PR source.